gpu-jobs.md

```diff
@@ -68,34 +79,17 @@ GPU Lab.
   </tr>
   <tr>
     <td>4</td>
-    <td>gpu2002 - gpu2005</td>
+    <td>gpulab2000 - gpulab2003</td>
     <td>8</td>
     <td>GeForce RTX 2080 Ti</td>
     <td>CentOS 7</td>
     <td>yes</td>
   </tr>
 </table>
 
-### Job types, runtimes, and per-user limitations
-
-**Jobs running in the CHTC GPU Lab have different time limits than normal CHTC job
-submissions**. 
-
-See the table below for the default runtime limits for normally submitted jobs
-
-{:.gtable}
-  | Job type | Maximum runtime | Per-user limitation |
-  | --- |
-  | Short | 12 hrs | 2/3 of CTHC GPU Lab GPUs |
-  | Medium | 24 hrs | 1/3 of CTHC GPU Lab GPUs |
-  | Long  | 7 days | 1 job with 1-2 GPUs |
-  | Pre-emptable (backfill) | None | None |
-
-For interactive jobs, the default time limit is 4 hours and only 1 GPU can be requested 
-at once. 
-
-These job types, runtimes, and per-user limitations are subject to change with
-short notice as the CHTC GPU Lab studies usage patterns.
+By opting-in to use the CHTC GPU Lab servers, you agree to be contacted by the
+project leaders occasionally to discuss your GPU computing and help improve the
+GPU Lab.
 
 ## B. Researcher Owned GPUs
```

Comment:
Is it a problem to have two "B" items?

Reply:
Maybe put "Researcher-Owned GPUs" (note the hyphen) lower, making sure to link from the main menu/linked list of topics on this page?

Reply:
If I put it lower, I feel like I'd be breaking up the page. Right now it's: 
- resources
- submit file
- software

Reply:
I could restructure to be: 
- shared use (GPU Lab)
- non-shared use
- software

The tricky piece there, is that some things apply across the board -- like, everybody needs to request GPUs. 

Maybe I'll try to draft that up and submit as a separate PR for you to look at.